### PR TITLE
 calc_pmv_ppd

### PR DESCRIPTION
**Pull request by Lu-Whale**
finished calc_pmv_ppd, modified v_relative_array in utilities.js

### CODE REVIEW

**Comment by szaldivar** [comment]
there are some parts that used `v_relative_array`, that is why the tests are failing, make sure to update them as well

**Comment by Lu-Whale** [comment]
> there are some parts that used `v_relative_array`, that is why the tests are failing, make sure to update them as well

updated the related test

File: docs/examples/calc_pmv_ppd.js [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+const tr_test = new Array(iterations).fill(23); // tr
+const rh_test = new Array(iterations).fill(40); //rh
+const met_test = new Array(iterations).fill(1.5); //met
+
+const v_rel_test = met_test.map((value) => v_relative(0.1, value)); //vr
+const clo_d_test_ASHRAE = met_test.map((value) => clo_dynamic(1, value)); //clo with the standard of ASHRAE
+const clo_d_test_ISO = met_test.map((value) => clo_dynamic(1, value, "ISO")); //clo with the standard of ISO
+
+// ASHRAE PMV
+const startAshrae = new Date().getTime();
```
Comment (szaldivar) [inline docs/examples/calc_pmv_ppd.js:88]: use `performance.now()`

**Review by szaldivar — CHANGES_REQUESTED** [review]

File: docs/examples/calc_pmv_ppd.js (hunk 1 above)
Comment (Lu-Whale) [reply docs/examples/calc_pmv_ppd.js]: done

**Review by szaldivar — APPROVED** [review]

**Pull request merged**